### DISCUSSION TITLE
Command line option for public server

### DISCUSSION THREAD

**Discussion by troykelly** [jamulussoftware/jamulus#2098] [Technical Support & Help] [answered]
Hi there,
I'm struggling to use the existing docker container https://hub.docker.com/r/grundic/jamulus
I can't work out which command-line option enables the server to register publically?
Any help would be much appreciated.

**Comment by gilgongo** [comment, marked as answer]
Hi @troykelly - server setup info for a public server is part of the "headless server" [setup here](https://jamulus.io/wiki/Server-Linux#running-in-public-mode). 

A list of command-line options for servers is [here](https://jamulus.io/wiki/Running-a-Server#command-line-options)

**Reply by troykelly** [reply]
Thank you!
